app/views/shared/_user_component_certificates.html.erb

```diff
@@ -6,7 +6,8 @@
        <%= t("user_journey.component_long_name.#{component.type}") %>
         <span class="component-supporting-text">
           <%= t 'user_journey.used_by' %>
```

Comment:
'**used by**' shouldn't be bold and it will be better if bring it together by: `<%= t 'user_journey.used_by', services:  component.services.map(&:name).join(', ') %>`

Reply:
So I tried bringing them together as suggested, but now the services are no longer visible. I'm not sure why

Reply:
did you add the placeholder to the locale?

Reply:
Sorry Jakub, I don't know what that means in this context 😕

Reply:
you need to change the locale if you're passing a variable in...
e.g. https://github.com/alphagov/verify-self-service/blob/75f775f373b4e94451a5a4b8efcbc3b578926e1c/config/locales/en.yml#L75

`used_by: used by` -> `used_by: "used by %{services}"`